inc/com/centreon/engine/service.hh

```diff
@@ -154,8 +172,7 @@ class                           service : public notifier {
   int                           retain_status_information;
   int                           retain_nonstatus_information;
   int                           obsess_over_service;
-  std::unordered_map<std::string, customvariable>
-                                custom_variables;
+  int                           problem_has_been_acknowledged;
```

Comment:
I think that was removed in a previous patch on service. I don't think we need it now.